zh_cn/documentation/ruby-from-other-languages/index.md

```diff
@@ -232,150 +229,128 @@ NoMethodError: private method `func' called for #<Test:0x342784>
 irb(main):026:0>
 {% endhighlight %}
 
-### Classes are open
+### 类是开放的
 
-Ruby classes are open. You can open them up, add to them, and change them at any time. Even core classes, like `Fixnum` or even `Object`, the parent of all objects. Ruby on Rails defines a bunch of methods for dealing with time on `Fixnum`. Watch:
+Ruby 的类是开放的。任何时候你都可以打开它进行增加和修改。即使像 `Fixnum` 这样的核心类，甚至是所有对象的父类 `Object` 类都可以打开。Ruby on Rails 在 `Fixnum` 类里定义了一大堆方法来处理时间问题。请看：
 
 {% highlight ruby %}
 class Fixnum
   def hours
-    self * 3600 # number of seconds in an hour
+    self * 3600 # 一小时有多少秒
   end
   alias hour hours
 end
 
-# 14 hours from 00:00 January 1st
-# (aka when you finally wake up ;)
+# 一月一号之后的 14 个小时
+# (也就是你醒来的时间)
 Time.mktime(2006, 01, 01) + 14.hours # => Sun Jan 01 14:00:00
 {% endhighlight %}
 
-### Funny method names
+### 有趣的方法名
```

Comment:
I thought this part is focused on method's name, not method itself. :cry: 
btw I'm totally noob. thanks for your help.